docs/source/pages/tutorials/exclude_include_checks.rst

```diff
@@ -0,0 +1,175 @@
+
+    .. note:: At the moment, Docker alternatives (e.g. podman) are not supported.
+
+
+    You also need to provide Macaron with a GitHub token through the ``GITHUB_TOKEN``  environment variable.
+
+    To obtain a GitHub Token:
+
+    * Go to ``GitHub settings`` → ``Developer Settings`` (at the bottom of the left side pane) → ``Personal Access Tokens`` → ``Fine-grained personal access tokens`` → ``Generate new token``. Give your token a name and an expiry period.
+    * Under ``"Repository access"``, choosing ``"Public Repositories (read-only)"`` should be good enough in most cases.
+
+    Now you should be good to run Macaron. For more details, see the documentation :ref:`here <prepare-github-token>`.
+
+------------------
+Motivating example
+------------------
+
+Let's say you want to run the analysis on `the micronaut-projects/micronaut-core Github repository <https://github.com/micronaut-projects/micronaut-core>`_ at the tag ``4.3.10``.
+
+Normally, this is how you would run Macaron:
+
+.. code-block:: shell
+
+  ./run_macaron.sh analyze -purl pkg:/github.com/micronaut-projects/micronaut-core@v4.3.10 --skip-deps
+
+By default, Macaron will run all of its checks. However, there can be checks that are not applicable to ``micronaut-projects/micronaut-core``.
+
+For example, the ``mcn_provenance_witness_level_one_1`` check (defined in :class:`ProvenanceWitnessL1Check <macaron.slsa_analyzer.checks.provenance_witness_l1_check.ProvenanceWitnessL1Check>`) is not applicable because ``micronaut-projects/micronaut-core`` doesn't generate and publish any `Witness provenance <https://github.com/testifysec/witness>`_. Therefore, we could exclude this check from running by performing the following steps.
+
+1. Prepare a ``defaults.ini`` configuration file with the following content.
+
+.. code-block:: ini
+
+    [analysis.checks]
+    exclude = mcn_provenance_witness_level_one_1
+    include = *
+
+In the ini configuration above:
+
+* The ``exclude`` option contains the check ID ``mcn_provenance_witness_level_one_1``. The check with this ID will be excluded.
+* The ``include`` option contains the `glob <https://docs.python.org/3/library/glob.html>`_ pattern ``*``. Checks whose ID matched this glob pattern will be included in the analysis. In this case, ``*`` matches all check IDs.
+
+2. Run Macaron analyze by providing the path to the custom ``defaults.ini`` configuration.
+
+.. code-block:: shell
+
+  ./run_macaron.sh -dp <path_to_defaults_ini> analyze -purl pkg:/github.com/micronaut-projects/micronaut-core@v4.3.10 --skip-deps
+
+In this run, the check ``mcn_provenance_witness_level_one_1`` mentioned above will not be run. After running the analyze command, we can view the data that Macaron has gathered about the ``micronaut-projects/micronaut-core`` repository at ``v4.3.10`` in an HTML report. Note that the result of the excluded check is not recorded in the Macaron HTML report (this also happens to JSON reports and the database).
```

Comment:
Suggestion: Avoiding multiple nearby uses of the word `run` under different meanings may be best.

```suggestion
This time, the check ``mcn_provenance_witness_level_one_1`` runs. After the `analyze` command finishes, we can view the data that Macaron has gathered about the ``micronaut-projects/micronaut-core`` repository at ``v4.3.10`` in an HTML report. Note that the result of the excluded check is not recorded in the HTML reports, JSON reports, or the database.
```

Reply:
I believe the first sentence should be `This time, the check ``mcn_provenance_witness_level_one_1`` doesn't run.`

Reply:
Fixed in https://github.com/oracle/macaron/pull/667/commits/f72fc8ed825545d5a193083b711cd6a0720a0465